include/libdnf/advisory/advisory_query.hpp

```diff
@@ -47,11 +47,10 @@ class AdvisoryQuery : public AdvisorySet {
 
     ~AdvisoryQuery();
 
-    struct NotSupportedCmpType : public RuntimeError {
-        using RuntimeError::RuntimeError;
+    struct NotSupportedCmpType : public Error {
+        NotSupportedCmpType();
```

Comment:
FWIW we agreed in a different PR (https://github.com/rpm-software-management/libdnf/pull/1365#discussion_r736502104) this should be an assertion, but that's for another PR (there's a lot of this in `PackageQuery`).

Reply:
OK. I'll remake it.

Reply:
@jrohel tbh. it just might be better to leave for a followup PR...